config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   root to: 'home#show'
+  get '/no-demo-night', to: 'home#no_demo_night', as: 'no_demo_night'
```

Comment:
this seems kind of hacky? maybe an errors#show action that gets params passed with it to show what the error is?